### DISCUSSION TITLE
GDB standalone and bundled fs-uae

### DISCUSSION THREAD

**Discussion by docbobo** [prb28/vscode-amiga-assembly#232] [Q&A]
Hey,

I'd like to use the bundled FS-UAE with gdb standalone and thought "How hard can that be?" 😉 So I tried starting it like this (on MacOS):

`/Users/docbobo/Library/Application Support/Code/User/globalStorage/prb28.amiga-assembly/downloads/vscode-amiga-assembly-binaries-@-1.7.0/vscode-amiga-assembly-binaries-osx/fs-uae --chip_memory=2048 --hard_drive_0=/Users/docbobo/Documents/Commodore Amiga/VSC/uae/dh0 --joystick_port_1=none --amiga_model=A1200 --remote_debugger=200 --use_remote_debugger=true --automatic_input_grab=0`, then trying to use m68k-amigaos-gdb (from ebbo) to attach to it.

However, this is the result:

```
(gdb) target remote <IP address>:6860
Remote debugging using <IP address>:6860
The target is not running (try extended-remote?)
```

Is this supposed to work in theory? Any hints?

**Comment by prb28** [comment]
Yes, it should work with a standard gdb, but I'm not sure it works well.

Did you consider using gdb toolkit?
It should work better with a standard gdb protocol.
